### ISSUE TITLE
Change the height of each row

### ISSUE THREAD

**Issue by mhdalalwan** [pavankataria/SwiftDataTables#10]
Hi, 
I was wondering if I could change the height of each row separately and also can i do word wrapping cause no matter how the sentence is long .. it will appear in one row.

Thanks in advance.

**Comment by pavankataria** [comment]
Hi there, thanks for posting. 
I'll need to think of a long term solution that would calculate the estimated height of each cell and actually look at the layout constraints. I welcome a pull request. 

What you could do in the mean time is to ensure your label's `numberOfLines` property is set to 0.  Then return your own height in the SwiftDataTables datasource method by calculating what the height of the label should be given a specific width.

**Comment by pavankataria** [comment]
> I was wondering if I could change the height of each row separately 

🎉 🎉 @mhdalalwan This is now available as of version 0.6.22 🎉 🎉 https://github.com/pavankataria/SwiftDataTables/releases/tag/0.6.22

You can now implement the `SwiftDataTableDelegate` and implement the following delegate method:

```Swift
    /// Specify custom heights for specific rows. A row height of 0 is valid and will be used.
    ///
    /// - Parameters:
    ///   - dataTable: SwiftDataTable
    ///   - index: the index of the row to specify a custom height for.
    /// - Returns: the desired height for the given row index
    @objc optional func dataTable(_ dataTable: SwiftDataTable, heightForRowAt index: Int) -> CGFloat
```

Let me know how it goes. 🎉
